pkg/operator/ceph/object/user/controller.go

```diff
@@ -705,25 +705,28 @@ func (r *ReconcileObjectStoreUser) objectStoreInitialized(cephObjectStoreUser *c
 	return errors.New("no rgw pod found")
 }
 
-func (r *ReconcileObjectStoreUser) getObjectStore(storeName string) (*cephv1.CephObjectStore, error) {
+func (r *ReconcileObjectStoreUser) getObjectStore(osu *cephv1.CephObjectStoreUser) (*cephv1.CephObjectStore, error) {
+	storeName := osu.Spec.Store
+	nsdName := client.ObjectKeyFromObject(osu)
```

Comment:
`nsName` is commonly used across rook for this
```suggestion
	nsName := client.ObjectKeyFromObject(osu)
```